src/screens/SearchScreen.spec.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { __esModule } from "@testing-library/jest-dom/dist/matchers";
+import { render, waitFor, screen } from "@testing-library/react";
+import SearchScreen from "./SearchScreen";
+import "@testing-library/jest-dom";
+import { BrowserRouter } from "react-router-dom";
+import { LoadScript } from "@react-google-maps/api";
+
+describe("Basic functionality", () => {
+  __esModule: true,
```

Comment:
This can be moved to jest.config.js I think

Reply:
Removed __esModule since its not needed here and testcases run perfectly without this config, hence jest.config.js not added.